decidim-admin/spec/system/space_admin_manages_global_moderations_spec.rb

```diff
@@ -24,6 +24,27 @@
     login_as user, scope: :user
   end
 
+  context "when the user can visualize the components" do
+    let!(:reportable) { create(:dummy_resource, component: current_component, title: { "en" => "Dummy Title" }) }
```

Comment:
Can we add some tags in the title so we test tags are stripped?
```suggestion
    let!(:reportable) { create(:dummy_resource, component: current_component, title: { "en" => "<p>Dummy Title</p>" }) }
```

Reply:
@tramuntanal , test added ...